src/main/java/com/softserve/model/User.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.Date;
 import java.util.List;
-
+import javax.persistence.*;
```

Comment:
avoid the star * when importing multiple classes from a same package, is not a good practice, import specific classes